components/email/src/init.spec.js

```diff
@@ -733,3 +733,188 @@ describe("Email: Toggle email of sender/recipient", () => {
     });
   });
 });
+
+describe("Should Render Reply Button And Dispatch Event When Clicked", () => {
+  it("Should Render Reply Button When Passed A Message", () => {
+    cy.visit("/components/email/src/examples/reply-message.html");
+
+    cy.get("nylas-email").then((component) => {
+      cy.get(component).should("exist");
+      cy.get(component).find("div.reply button").should("exist");
+    });
+  });
+
+  it("Should Render Reply Button When Passed A Thread", () => {
+    cy.visit("/components/email/src/examples/reply-thread.html");
+
+    cy.get("nylas-email").then((component) => {
+      cy.get(component).should("exist");
+      cy.get(component).find("div.reply button").should("exist");
+    });
+  });
+
+  it("Should Render Reply Button When Passed A Message ID", () => {
+    cy.intercept(
+      {
+        method: "GET",
+        url: "**/messages/d0byfc378l2728z35pax362ho",
+      },
+      {
+        fixture: "email/messages/id.json",
+      },
+    ).as("messageRequest");
+
+    cy.visit("/components/email/src/examples/reply-message_id.html");
+    cy.wait("@messageRequest");
+
+    cy.get("nylas-email").then((component) => {
+      cy.get(component).should("exist");
+      cy.get(component).find("div.reply button").should("exist");
+    });
+  });
+
+  it("Should Render Reply Button When Passed A Thread ID", () => {
+    cy.intercept(
+      {
+        method: "GET",
+        url: "**/threads/e2k5xktxejdok7d8x28ljf44d?view=expanded",
+      },
+      {
+        fixture: "email/threads/id.json",
+      },
+    ).as("threadRequest");
+
+    cy.intercept(
+      {
+        method: "GET",
+        url: "**/messages/*",
+      },
+      {
+        fixture: "email/messages/id.json",
+      },
+    );
+
+    cy.visit("/components/email/src/examples/reply-thread_id.html");
+    cy.wait("@threadRequest");
+
+    cy.get("nylas-email").then((component) => {
+      cy.get(component).should("exist");
+      cy.get(component).find("div.reply button").should("exist");
+    });
+  });
```

Comment:
Since these are such simple test cases, you should alias the component and use `cy.get("@email").invoke("attr", <prop name>, <prop value>` instead of creating new html files.

```
// alias component in beforeEach
  beforeEach(() => {
    cy.visit("/components/email/src/index.html");

    cy.get("nylas-email").as("email");
  });

// passing message id
cy.get("@email").invoke("attr", "message_id", "d0byfc378l2728z35pax362ho");

// passing message
cy.get("nylas-email").then((component) => {
component.message = SAMPLE_MESSAGE; // add message var from reply-message.html to this file 

// passing thread 
cy.get("nylas-email").then((component) => {
component.thread = SAMPLE_THREAD;

// passing thread id
cy.get("@email").invoke("attr", "thread_id", "e2k5xktxejdok7d8x28ljf44d");

```